python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -29,8 +29,24 @@ router bgp {{ router_bgp.as }}
 {%     elif router_bgp.bgp.default.ipv4_unicast_transport_ipv6 is arista.avd.defined(false) %}
    no bgp default ipv4-unicast transport ipv6
 {%     endif %}
-{%     if router_bgp.timers is arista.avd.defined %}
-   timers bgp {{ router_bgp.timers }}
+{%     if router_bgp.timers.keepalive_time is arista.avd.defined and
+        (router_bgp.timers.hold_time is arista.avd.defined or router_bgp.timers.hold_forever is arista.avd.defined) %}
+{%         set timers_bgp_cli = "timers bgp " ~ router_bgp.timers.keepalive_time %}
+{%         if router_bgp.timers.hold_time is arista.avd.defined %}
```

Comment:
Where are you implementing hold_forever?

Reply:
its in the else part, need tot add the check for "true" (updated it)

Reply:
Updated the schema